src/main/resources/application.yaml

```diff
@@ -5,3 +5,24 @@ logging:
   level.org.springframework.web: WARN
 server:
   port: 8080
+application:
+  elastic:
+    username: ${ELASTIC_USERNAME}
```

Comment:
Vi mangler env vars for staging i deploy

Reply:
@valosnah fdk-search-service trenger vel også en egen bruker i elastic?